docs/rulebook.md

```diff
@@ -0,0 +1,100 @@
+# Rulebook / constitution
+
+This is a small document that defines how this community is working.
+
+## Project role
+
+Provide a shared development environment and infrastructure for the Nix and
+NixOS ecosystem, strictly for non-profit purposes.
+
+## Members
+
+There are currently two classes of members in the project:
+
+* Community members - participants of our community.
+* Administrators - members who own and administer the project.
+
+## Community members
+
+### Team
+
+Anybody who contributes code to one of our project is considered a community
+member.
```

Comment:
I don't see how this definition is feasible if community members have some sort of voting rights?

Reply:
Generally, open-source authority stems from a combination of several factors; contribution, longevity, and dependability.

1) Have they contributed value?
2) Have they shown a long-term interest in the project?
3) Have they been responsive to the needs of the project?

Only basing this on any single of the above factors leads to odd situations.

Reply:
There are a lot of dimensions that make a good contributor. Sometimes the user only hangs out on Matrix and helps users out. Or they submitted one patch that has an incredible impact. Or they help maintain a good spirit in the community just by who they are. Or ...

Reversely, some of the value of contribution may decay over time. Should users that submitted a patch 10 years ago still have voting rights?

It's hard to find a single metric that can be used as an inclusion criterion. And those metrics can probably be gamed pretty easily.

I think what's important is that you can feel like you have a voice when participating in this community. And that you can be confident that the project is handled well.

Reply:
How about the following criteria:

* Has at least 10 contributions to projects that are in the org
* The last contribution is younger than 2 years old

Reply:
fixed in ba214bd0cfca7ab5e2df4a61a0c1b4de0446e2bb